bootstrap.bat

```diff
@@ -1,6 +0,0 @@
-@echo off
```

Comment:
I do not know where to put this question, so excuse me please. With the cleaning from old building process scripts like `bakefile` and others is it also desired to delete `configure` and `configure.ac` and `Makefile.in`?